.github/workflows/build-image-frontend.yml

```diff
@@ -51,6 +51,41 @@ jobs:
             -   name: Checkout
                 uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
 
+            -   name: Setup Node
+                uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+                with:
+                    node-version: 20
+                    registry-url: https://registry.npmjs.org/
+
+            -   name: Get npm version
+                id: npm-tag
+                uses: martinbeentjes/npm-get-version-action@3cf273023a0dda27efcd3164bdfb51908dd46a5b # v1.3.1
+
+            -   name: Install Dependencies
+                working-directory: ./frontend
+                run: npm install
+
+            #substring about license
```

Comment:
the comment is a bit weird, maybe Replace for legal notice in about page

Reply:
corrected.